### ISSUE TITLE
Factor pi missing in `compute_cheby_coeff`

### ISSUE THREAD

**Issue by kieranricardo** [epfl-lts2/pygsp#70]
It appears that a factor of `pi` is missing in the numerical integration in https://github.com/epfl-lts2/pygsp/blob/d78095438a14a74d2195d4600fc00f8f1115005f/pygsp/filters/approximations.py#L52 compared Eq 8 https://arxiv.org/pdf/1105.1891.pdf throwing off the Chebyshev coefficients by a factor of `pi`.

**Comment by nperraud** [comment]
Hi thanks for reporting that.
@mdeff , do you know if this is intentional or a mistake. I have the feeling we knew about that... If it is intentional, we should document it.

**Comment by mdeff** [comment]
Indeed, we use [`2/n` as a factor](https://github.com/epfl-lts2/pygsp/blob/70f47eaf5d4b521f3247c4f5f10e7df1f39b2b47/pygsp/filters/approximations.py#L52) instead of `2/pi`. As for #74, the definition of the coefficients came from the [matlab implementation](https://github.com/epfl-lts2/gspbox/blob/a7d9aac5e239f1bcb37a9bb09998cc161be2732f/utils/gsp_cheby_coeff.m#L102). I believe that is intentional and correct, as the integration over `pi` is approximated by a summation over `n` evaluation points. Do you agree @nperraud?

In my [overhaul of the Chebyshev approximation](https://github.com/epfl-lts2/pygsp/blob/polynomial-approximations/pygsp/filters/approximations.py) (in the `polynomial-approximations` branch), I used `2/n` and [documented the definition](https://github.com/epfl-lts2/pygsp/blob/c0868318f26572ebf5702425c82e7b8bce3f77a8/pygsp/filters/approximations.py#L101). I could add that the summation is an approximation of the integral.
